awspub/sns.py

```diff
@@ -38,7 +39,34 @@ def conf(self) -> List[Dict[str, Any]]:
         """
         return self._ctx.conf["images"][self._image_name]["sns"]
 
-    def _get_topic_arn(self, topic_name: str) -> str:
+    def _sns_regions(self, topic_config: Dict[Any, Any]) -> List[str]:
+        """
```

Comment:
this seems to be a copy of `_image_regions()` from image.py. please extract the logic (eg. into common.py) and reuse it from there.

Reply:
Moved logic to `common.py` and updated test, image.py and sns.py files in the last commit.